code/script.js

```diff
@@ -1,51 +1,117 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
 
-// If you need any global variables that you can use across different functions, declare them here:
+let botQuestion = 1
 
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
 
-// Declare your functions after this comment
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
 
```

Comment:
Making the answers from the bot and the user into variables makes the code shorter when we need it further down in the code 👍